src/styles/index.css

```diff
@@ -71,4 +72,25 @@ body {
   box-shadow: 0px 6px 0px theme('colors.black');
 }
 
+@keyframes float {
```

Comment:
Cool!

I would move this to a separate CSS module though since we only use it on one page.

Reply:
i made aseparate emojiPicker.module.css inside style folder, is that what you had in mind?